DOC add a point about org bans due to AI (#32254)
scikit-learn/scikit-learn: CODE_OF_CONDUCT.md

We abide by the principles of openness, respect, and consideration of others of
the Python Software Foundation: https://www.python.org/psf/codeofconduct/

# Low Quality and AI Generated Contributions Policy

Due to the burden put on maintainers, users submitting multiple low quality pull
requests, or AI generated comments, reviews, issues, or pull requests, where the

> user does not show a good understanding of what they are posting, might be banned

Reviewer: Maybe we could include poor etiquette as a reason (e.g., trying to 'take' an issue where it hasn't even been decided what we should do, or when others have already asked to work on it, 'spamming' several issues to work on, opening duplicate PRs etc). I don't know what the best phrase/description would be, but I find it often goes hand in hand with AI generated or low quality PRs.

from the organisation.

For more context, you can check out this blog post on [
The Cost of AI in Open Source Maintenance
](https://adrin.info/the-cost-of-ai-in-open-source-maintenance.html).

If this happens to you and you believe it's been a mistake, you can reach us on

> `coc@scikit-learn.org`.

Reviewer 2: No we don't but I can create one if we prepare a list.

> `coc@scikit-learn.org`.

Author: I'm okay being on the list (don't have to though), but we need more than one person there. @lucyleeow would you mind being there? Hopefully it'll be a very low traffic thing.

We abide by the principles of openness, respect, and consideration of others of
the Python Software Foundation: https://www.python.org/psf/codeofconduct/

# Low Quality and AI Generated Contributions Policy

Due to the burden put on maintainers, users submitting multiple low quality pull
requests, or AI generated comments, reviews, issues, or pull requests, where the
user does not show a good understanding of what they are posting, might be banned
from the organisation. Some examples of poor etiquette are:

> - Opening a PR for issues where they are not triaged yet and the "triage" label is not

Reviewer 3: ```suggestion
- Opening a PR for issues which are not yet triaged and the "triage" label is not
```
